### ISSUE TITLE
[Bug?] Setting Options in Edit/Practice Requires 2p Controls

### ISSUE THREAD

**Issue by androoP** [TeamRizu/OutFox#467]
If you want to set a speed mod (Or any option) in practice mode, you **must** have 1p and 2p controls configured. 

Both players must highlight "Exit" to save settings and exit the menu, but if no 2P controls are set, you can't set/exit. 

Note: This happens regardless if "Show Both Players" is set to No.
![Inked2021-12-29 10_49_17-Project OutFox 5 3 Alpha](https://user-images.githubusercontent.com/9681605/147680486-c50e7a29-bc5b-4a4c-9e7f-a63f4e180f7f.jpg)

**Comment by JoseVarelaP** [comment]
Yeah, this is a bug; more like an oversight of mine. Since the editor can now perform playback with more than one player, ScreenOptions (The class that manages the listing of the options) will generate a list that loads both players at the same time.

This might send me to a rabbit hole to create a new set of options to prevent this from happening, given it's locked to that player configuration.

**Comment by JoseVarelaP** [comment]
This was fixed back in 2024. Closing.
